### PR TITLE
Added new function getFriendlyDate

### PR DESCRIPTION
**Pull request by qodo**
This compares two DateTime values and then formats into a human friendly value as raised in issue #80 and uses the same format as `describeInterval` function.

### CODE REVIEW

File: code/Calendar.php [hunk 1]
```diff
@@ -876,6 +876,197 @@ public static function getDeltaDays($dateEnd, $dateStart = null)
...
+                        } else {
+                            $parts[] = Unit::format($diff->s, 'duration/second', $width, $locale);
+                        }
+                        break;
+                }
+            }
+
+        }
+
+        // print_r($parts);
```
Comment (Remo) [inline code/Calendar.php]: can you please remove that?

Comment (qodo) [inline code/Calendar.php]: Sorry! This has been removed.

**Comment by mlocati** [comment]
Could you add some tests (like we already have in `tests/Calendar/CalendarTest.php` - for the `describeInterval` method)?

**Comment by mlocati** [comment]
@qodo I'm going to close this because of inactivity. Fell free to reopen it, provided that you add test cases.

**Pull request closed without merging**